README.md

```diff
@@ -1 +1,26 @@
+# computer-portfolio
+> Computer-portfolio
+
+# this project holds all part of computer portfolio
+- header
+- body
+- about my self
+- contact me
+
+## Author 
+- Email [Girma Tarekegn] (https://email.com/tarekegngirma21@gmail.com)
+- GitHub [@girmatarekegn](https://github.com/girma336)
+- Twitter: [@GGt7045](https://twitter.com/GGt7045)
+
+## build with 
+- Linters hrml-css
```

Comment:
- [ ] Please include the Link to the deployed version of your portfolio. Currently, I am unable to find it anywhere here 😃